### PR TITLE
Add PowerShell Dockerfile

### PR DESCRIPTION
**Pull request by tylerl0706**
This adds the PowerShell Language Worker Dockerfile.

It grabs the nupkg from the myget feed
Unzips it and places it in the host's workers folder

### CODE REVIEW

File: host/2.0/stretch/amd64/powershell.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+FROM azure-functions/base:${BASE_IMAGE_TAG} AS functions-base
+ARG BASE_IMAGE_TAG
+ARG WORKER_TAG
+
+# Add the nupkg to the container and unzip it
+ADD https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.Azure.Functions.PowerShellWorker/${WORKER_TAG} PowerShellWorker.nupkg
+RUN apt-get update && apt-get install -y unzip && unzip -q PowerShellWorker.nupkg
+
+# Copy the powershell worker to the workers folder
+FROM azure-functions/base:${BASE_IMAGE_TAG}
```
Comment (daxian-dbw) [inline host/2.0/stretch/amd64/powershell.Dockerfile]: Alright, you already changed to use multi-stage build 😄

Comment (tylerl0706) [reply host/2.0/stretch/amd64/powershell.Dockerfile]: Huge credit goes to @TravisEz13 for showing me 😊

**Comment by tylerl0706** [comment]
I'm a little confused by the merge conflict because I don't know what `mesh` is. I'm hoping @ahmelsayed has the answer on what should the result of conflict be.

**Comment by ahmelsayed** [comment]
We will need another PR here https://github.com/Microsoft/mcr/tree/master/teams/azurefunctions to onboard this image on MCR. I can do that part.

**Comment by tylerl0706** [comment]
Thanks @ahmelsayed!

**Pull request merged**